src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Vault/SetAzureRmRecoveryServicesVaultProperty.cs

```diff
@@ -31,11 +31,16 @@ public class SetAzureRmRecoveryServicesVaultProperty : RSBackupVaultCmdletBase
         internal const string AzureRSVaultSoftDelteParameterSet = "AzureRSVaultSoftDelteParameterSet";
         internal const string AzureRSVaultCMKParameterSet = "AzureRSVaultCMKParameterSet";
 
-        [Parameter(Mandatory = true, ValueFromPipeline = false, ParameterSetName = AzureRSVaultSoftDelteParameterSet)]
+        [Parameter(Mandatory = false, ValueFromPipeline = false, ParameterSetName = AzureRSVaultSoftDelteParameterSet)]
         [ValidateNotNullOrEmpty]
         [ValidateSet("Enable", "Disable")]
         public string SoftDeleteFeatureState { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipeline = false, ParameterSetName = AzureRSVaultSoftDelteParameterSet)]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("Enable", "Disable")]
+        public string HybridBackupSecurityFeature { get; set; }
```

Comment:
Have you submit a design review for this cmdlet? Please submit one in https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues. In our best design practice, we would recommend to name HybridBackupSecurityFeature as EnableHybridBackupSecurityFeature and set its type as bool?.
- If true, it means enable;
- If false, it means disable;
- if null, it means no change.

Reply:
We have a cmdlet-review-pr for this: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1131

However, we can keep it bool as you suggested. But switch parameter is not sufficient for this case, hence will keep the type as bool

Reply:
As you mentioned in cmdlet review,  by default security features are always enabled. So we would like to recommend DisableHybridBackupSecurityFeature here. And its type should be nullable bool. Thanks.